src/nspawn/nspawn-register.c

```diff
@@ -40,6 +41,12 @@ static int append_machine_properties(
                                   "char-pts", "rw");
         if (r < 0)
                 return bus_log_create_error(r);
+        if (enable_fuse) {
+                r = sd_bus_message_append(m, "(sv)", "DeviceAllow", "a(ss)", 1,
+                                          "/dev/fuse", "rw");
```

Comment:
Here also, "rw" -> "rwm"

Reply:
Really? Doesn't `copy_devnodes()` run before this scope is set up?

Reply:
Hm, indeed, I thought that because /dev/net/tun in the above has "m". I am not sure here "m" is really necessary.

Reply:
hmm, i guess we should drop "m" from 7dev/net/tun if we create the device node ourselves